t/t-track.sh

```diff
@@ -375,9 +375,11 @@ begin_test "track blocklisted files by name"
 
   touch .gitattributes
   git add .gitattributes
+  git commit -m 'Initial commit'
 
   git lfs track .gitattributes 2>&1 > track.log
   grep "Pattern '.gitattributes' matches forbidden file '.gitattributes'" track.log
+  [ -z "$(git st --porcelain | grep -v '^??')" ]
```

Comment:
Both here and in the test below, is this an alias for `git status`?  It may not be something that's installed locally.

I notice this succeeds as a test in CI and on my system because the pipe into `grep` means the non-zero exit code from `git st`, reporting `'st' is not a git command`, doesn't trigger an immediate failure (thanks, `set -e` in Bash!), and since there's nothing on stdout, the `grep -v` then returns nothing and the `-z` passes.